src/App.jsx

```diff
@@ -1,18 +1,25 @@
-import { Routes, Route } from "react-router-dom"
+import { useContext } from "react"
+import { Routes, Route, Navigate} from "react-router-dom"
 import Login from "./components/Login"
 import Main from "./components/Main"
 import Header from "./components/Header"
-import Footer from "./components/Footer"
```

Comment:
Hebben we geen Footer meer?

Reply:
Klopt, die had Marthe erin gezet in het begin, maar niet nodig gezien hoe we onze pagina overview willen.
Ik heb hem in eerste instantie geprobeerd te behouden, maar na 4 uur css geklooi om het op de juiste plek te krijgen bleek dat we hem überhaupt nooit gepland hadden.